enterprise/frontend/src/metabase-enterprise/sandboxes/actions.js

```diff
@@ -75,6 +78,27 @@ const groupTableAccessPolicies = handleActions(
         };
       },
     },
+    [UPDATE_DATA_PERMISSION]: {
```

Comment:
We should check and see if there was a bug logged for the existing wrong behavior here. Maybe there is another one we can close 🙌